ui/app/styles/components/linked-block.scss

```diff
@@ -23,3 +23,9 @@
   grid-row: 1;
   align-self: center;
 }
+
+.linked-block-title {
+  display: grid;
+  grid-template-columns: minmax(0, 28px) minmax(0, 1fr);
```

Comment:
There is a helper file called `flexbox-and-grid.scss` there's a chance you could use the helpers here instead of adding to the component. Our general rule of thumb with custom scss (e.g. we can't swap with an HDS component, which we can't here) is:
1. use a helper first
2. if it can't be done via a helper double check that it can't be achieved in one of our styles/core/xx.scss files (most of the time it can't)
3. and last resort add to the component .scss, which in this case is probably the best option.